features/SPS002_Correct_spatial_breakdown.feature

```diff
@@ -0,0 +1,11 @@
+@implementer-agreement
+@SPS
+Feature: SPS002 - Correct spatial breakdown
+The rule verifies that spatial elements are aggregated as per the Spatial Composition Table.
+The possible allowed breakdown can be found in the csv file in the folder 'features/resources/spatial_CompositionTable.csv'
+
+  Scenario: Agreement on each IfcSpatialElement being aggregated to other IfcSpatialStructureElement as per spatial composition table.
+
+      Given A file with Schema Identifier "IFC4X3_TC1" or "IFC4X3_ADD1" or "IFC4X3"
+      And An IfcSpatialElement
+      Then Each IfcSpatialElement must be aggregated to other IfcSpatialStructureElement as per spatial_CompositionTable.csv
```

Comment:
Isn't this rule applicable to all version of the schema? @evandroAlfieri

Reply:
Very much. Please @JTurbakiewicz extend the Given with "IFC2X3" and "IFC4". Also nice if you can add a couple of test files to corroborate this behavior. Thanks